ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -343,6 +348,23 @@ public boolean onConsoleMessage(ConsoleMessage message) {
       }
 
       @Override
+      public void onProgressChanged(WebView view, int newProgress) {
+        super.onProgressChanged(view, newProgress);
+
+        if(newProgress != 100) {
+          return;
+        }
+
+        WritableMap event = webView.createWebViewEvent(view.getUrl());
+        dispatchEvent(
+          view,
+          new TopLoadingFinishEvent(
+            view.getId(),
+            event));
+      }
+    });
```

Comment:
I think this block should be closed much further down - removing this line fixed issues with the method definitions below.